### DISCUSSION TITLE
[Feature] Manually tag people in photos when face not detected

### DISCUSSION THREAD

**Discussion by johnpc** [immich-app/immich#5333] [Feature Request]
### The feature

I have several photos with people in them where the face was not detected. Sometimes this is because it's the back of their head or they've got their head turned at a funny angle, or an object or other person is obstructing the face. However other times it's a mystery as the face is clear enough. However, since the ai doesn't detect any face, it is impossible to classify it as a photo of that person.

I would like there to be a way to click or tap on the person and manually tag them in the photo in this case.

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by johnpc** [comment]
This feels related to https://github.com/immich-app/immich/discussions/5321 but not exactly the same

**Comment by martabal** [comment]
It would be cool, but if a person has only manually tagged faces, the microservices can't associate new faces with that new person

**Reply by johnpc** [reply]
I don't see why microservices would have to get involved, it would just allow manual tags to appear alongside AI recognized photos in the "people" view

**Reply by martabal** [reply]
My point is that for a person with only manually tagged faces, if you upload a photo of that person recognized by the machine-learning the server will create a new person and you'll have duplicate people

**Reply by johnpc** [reply]
I see where there would probably be some additional logic needed around merging people. 

To be honest, I'd also not mind if it was only possible to tag existing people. In my use case there are not any people that I want to manually tag that don't exist at all.

**Reply by fordp2002** [reply]
> My point is that for a person with only manually tagged faces, if you upload a photo of that person recognized by the machine-learning the server will create a new person and you'll have duplicate people

This will not be an issue as it will suggest you merge them when you name the first machine-learned face.

**Comment by bo0tzz** [comment]
Duplicate of https://github.com/immich-app/immich/discussions/2472#discussioncomment-6850486
